docs/source/installation.rst

```diff
@@ -157,26 +157,102 @@ error log (in Apache on Debian/Ubuntu, for example, this is ``/var/log/apache2/e
 --------------------
 Deployment on Docker
 --------------------
-
 It is also possible to deploy the server using Docker.
 
-First build the Docker Image.  From the ``deploy`` folder run:
+First, you need an environment running the docker daemon. For non-production use, we recommend `boot2docker <http://boot2docker.io/>`_. For production use you should install docker on a stable linux distro.
+`Platform specific Docker installation instructions <https://docs.docker.com/installation/) (OS X and Windows are instructions for boot2docker>`_
+
+**Local Dataset Mounted as Volume**
+
+If you already have a dataset on your machine, you can download and deploy the apache server in one command: 
 
 .. code-block:: bash
 
-  $ docker build .
+  $ docker run -e GA4GH_DATA_SOURCE=/data -v /my/ga4gh_data/:/data:ro -itd -p 8000:80 --name ga4gh_server afirth/ga4gh_apache_server:prod
+
+Replace ``/my/ga4gh_data/`` with the path to your data.
+
+**Demo Dataset Inside Container**
 
-Find the ``id`` of the recently built image by running
+If you do not have a dataset yet, you can deploy a container which includes the demo data:
 
 .. code-block:: bash
 
-  $ docker images
+  $ docker run -itd -p 8000:80 --name ga4gh_demo afirth/ga4gh_server_apache:demo
 
-Use that ``id`` to launch the server
+**Ports**
+
+This will run the docker container in the background, and translate calls from your host environment
+port 8000 to the docker container port 80. At that point you should be able to access it like a normal website, albeit on port 8000.
+Running in boot2docker, you will need to forward the port from the boot2docker vm to the host.
```

Comment:
It would be nice to link to boot2docker here, in case the reader wasn't paying attention earlier.

Also, s/vm/VM